src/Components/Admins/Form/index.js

```diff
@@ -0,0 +1,259 @@
+    });
+  };
+
+  const handleClose = () => {
+    setIsOpen(!isOpen);
+    history.push('/admins');
+  };
+
+  return (
+    <section className={styles.sectionForm}>
+      <Modal
+        title={responseModal.title}
+        desc={responseModal.description}
+        isOpen={isOpen}
+        confirmModal={responseModal.isConfirm}
+        handleClose={() => handleClose()}
+      />
+      <form className={styles.form} onSubmit={onSubmit}>
+        <div className={styles.block}>
+          <div className={styles.firstPart}>
+            <TextInput
+              labelName="First name"
+              inputType="text"
+              inputName="firstName"
+              id="firstName"
+              text={formChange.firstName}
+              changeAction={(e) => onChangeInput(e)}
+            />
+          </div>
+          <div className={styles.firstPart}>
+            <TextInput
+              labelName="Last name"
+              inputType="text"
+              inputName="lastName"
+              id="lastName"
+              text={formChange.lastName}
+              changeAction={(e) => onChangeInput(e)}
+            />
+          </div>
+          <div className={styles.firstPart}>
+            <TextInput
+              labelName="DNI"
+              inputType="text"
+              inputName="dni"
+              id="dni"
+              text={formChange.dni}
+              changeAction={(e) => onChangeInput(e)}
+            />
+          </div>
+          <div className={styles.firstPart}>
+            <TextInput
+              labelName="Phone"
+              inputType="text"
+              inputName="phone"
+              id="phone"
+              text={formChange.phone}
+              changeAction={(e) => onChangeInput(e)}
+            />
+          </div>
+          <div className={styles.firstPart}>
+            <TextInput
+              labelName="Email"
+              inputType="text"
+              inputName="email"
+              id="email"
+              text={formChange.email}
+              changeAction={(e) => onChangeInput(e)}
+            />
+          </div>
+          <div className={styles.firstPart}>
+            <TextInput
+              labelName="City"
+              inputType="text"
+              inputName="city"
+              id="city"
+              text={formChange.city}
+              changeAction={(e) => onChangeInput(e)}
+            />
+          </div>
+          <div className={styles.firstPart}>
+            <TextInput
+              labelName="Password"
+              inputType="password"
+              inputName="password"
+              id="password"
+              text={formChange.password}
+              changeAction={(e) => onChangeInput(e)}
+            />
+          </div>
+        </div>
+        <div className={styles.btnContainer}>
+          <Button text="Cancel" clickAction={cancel} type="cancel" />
+          <Button text="Confirm" clickAction={onSubmit} type="Create" />
```

Comment:
We are using Add instead of confirm
```suggestion
          <Button text="Add" clickAction={onSubmit} type="Create" />
```

Reply:
Im validaiting by the id if appears one button or the other.

Reply:
it's perfect in that way! but the text in the button should change between save when you update and add when you create